src/app/store.ts

```diff
@@ -1,6 +1,15 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
The import statement uses `combineSlices`, which is incorrect. It should be `combineReducers` from `@reduxjs/toolkit` to properly combine the reducers.